firmware/src/sys/board.hpp

```diff
@@ -58,6 +58,9 @@ void resetWatchdog();
 
 void setStatusLed(bool state);
 void setCanLed(bool state);
+void DutycycleCounterPP();
+void DutycycleCounterMM();
+signed DutycycleCounterRead();
```

Comment:
Please follow coding conventions! Functions are named in `camelCase`, variables in `snake_case`, constants in `PascalCase`. What are these `MM` and `PP`, what do they mean? Getters are named like `getDutyCycleCounter()`.

Why the counter variable is signed? However, all of this is not the main issue. The main issue is why the duty cycle counter defined in the `board` namespace? Board API is intended to contain hardware-specific drivers and API; application-level stuff does not belong there.